### PR TITLE
Redesign header

### PR DESCRIPTION
**Pull request by josepjaume**
## What and why?

This redesigns the header to follow the latest design.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -79,6 +79,8 @@ gem 'i18n_yaml_csv', github: 'josepjaume/i18n_yaml_csv'
 
 gem 'draper'
 
+gem 'bourbon'
```
Comment (beagleknight) [inline Gemfile:82]: Is `compass` dead? :smile:

Comment (josepjaume) [inline Gemfile:82]: `compass` is legacy!! :dancer:

File: app/assets/javascripts/components/meetings_map.es6.jsx [hunk 2]
```diff
@@ -55,6 +55,15 @@ class MeetingsMap extends React.Component {
               lng: meeting.address_longitude,
             },
             animation: google.maps.Animation.DROP,
+            icon: {
+              path: "M25 0c-8.284 0-15 6.656-15 14.866 0 8.211 15 35.135 15 35.135s15-26.924 15-35.135c0-8.21-6.716-14.866-15-14.866zm-.049 19.312c-2.557 0-4.629-2.055-4.629-4.588 0-2.535 2.072-4.589 4.629-4.589 2.559 0 4.631 2.054 4.631 4.589 0 2.533-2.072 4.588-4.631 4.588z",
```
Comment (beagleknight) [inline app/assets/javascripts/components/meetings_map.es6.jsx:59]: SVG POWER!

**Comment by beagleknight** [comment]
Thumbs up man! :+1: (circleci is complaining about a random error, don't worry!)

**Pull request merged**